MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -1,72 +1,208 @@
 import UIKit
 
-final class MovieQuizViewController: UIViewController {
-    // MARK: - Lifecycle
-    override func viewDidLoad() {
-        super.viewDidLoad()
+final class MovieQuizViewController: UIViewController, QuestionFactoryDelegate {
+    @IBOutlet private var moviePoster: UIImageView!
+    @IBOutlet private var questionForUser: UILabel!
+    @IBOutlet private var questionNumber: UILabel!
+    @IBOutlet private var noButton: UIButton!
+    @IBOutlet private var yesButton: UIButton!
+    @IBOutlet private var activityIndicator: UIActivityIndicatorView!
+    @IBOutlet weak var movieTitle: UILabel!
```

Comment:
```suggestion
    @IBOutlet private weak var movieTitle: UILabel!
```
Всегда старайтесь использовать наиболее закрытый модификатор доступа